### DISCUSSION TITLE
Zusammenstellung der Züge in der Fahrzeugauswahl für eine Ausschreibung

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#200] [Ideas]
D.h. Kuppeln (und Entkuppeln) wären auch möglich, wenn das Fahrzeug für eine Ausschreibung ausgewählt wird.
Das würde es deutlich einfacher machen, einen Fahrzeugpool flexibel einzusetzen.

Es wäre dabei auch denkbar, dass zumindest das Kuppeln nach Einsatzende rückgängig gemacht wird (das Trennen von Zügen wäre natürlich komplizierter, weil der zurückbleibende Teil inzwischen gesperrt sein könnte)

**Comment by marhei** [comment]
Das steht schon länger auf die Liste, leider ist das ganze recht komplex…
